homeassistant/components/aemet/weather_update_coordinator.py

```diff
@@ -95,15 +97,15 @@ def format_condition(condition: str) -> str:
     return condition
 
 
-def format_float(value) -> float:
+def format_float(value) -> float | None:
```

Comment:
Please change the methods where format_float is used, to ensure the check is checking something.